src/components/CourseTable/CourseTable.test.jsx

```diff
@@ -9,9 +9,28 @@ describe('CourseTable', () => {
     expect(component).toMatchSnapshot();
   });
 
-  it('hides table and button when not whitelisted', () => {
-    process.env.ORG_WHITELIST = ['fake'];
-    const component = shallow(<CourseTable />);
+  it('hides table and button when blacklisted', () => {
+    const publisherUserInfo = { organizations: [{ key: 'fake1', name: 'fake_name1' }] };
+    const component = shallow(<CourseTable publisherUserInfo={publisherUserInfo} />);
+    expect(component).toMatchSnapshot();
```

Comment:
Just want to confirm that the snapshot for this actually does not have the table showing.

That was an issue I was having in my PR, but it looks like you're doing it correctly with the setupTest.js file below.

Reply:
yes, that snapshot does not have the table 👍